test/Prompt/DontWrap.C

```diff
@@ -172,7 +172,7 @@ const Test2::Nested::Value::type & Test2::Nested::reference() const {
 Test2::Nested Nest2(80);
 // CHECK: Nested::Nested(160)
 Nest2.simpleAdd(3)
-// CHECK: (Test2::Nested::Value::type) 83
+// CHECK: (int) 83
```

Comment:
But the world isn't perfect and a lot of this is subjective...to me reduction of builtin types is better, though I do see that it is not the best 100% of the time. Without it many tests for Windows will have to be re-written, and what you are suggesting increases that to many more.

I think this thread got it to a point that is reasonably nicer than it was before, and the PR as a whole simplifies some spaghetti to make moving towards the _best_ solution easier.